statsmodels/genmod/tests/test_glm_weights.py

```diff
@@ -197,12 +207,12 @@ def setup_class(cls):
         cls.res2 = res_stata.results_poisson_pweight_nonrobust
 
     @pytest.mark.xfail(reason='Known to fail')
-    def test_basic(cls):
-        super(cls, TestGlmPoissonPwNr).test_basic(cls)
+    def test_basic(self):
```

Comment:
Well, that's embarrassing for the person who wrote that...

Oh wait...

Reply:
That's the tricky thing with blanket exceptions or xfails.
I tried to see how bad it is if we run the test, which ended up in #4234, after fixing the super call.
(I thought prob_weights will be "obvious" but it looks more like back to the drawing board)